src/EmployeeManagement/EmployeeManagement.Application/DTO/GetAllEmployeesDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EmployeeManagement.Application.DTO
+{
+    public class GetAllEmployeesDTO
```

Comment:
Can you please consider a more generic name, As 'GetAllEmployeesDTO' suggests is a specific name for retrieving all employees, but this DTO could be reused for other purposes as well.

Reply:
renamed to GetEmployeesDTO